third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -257,6 +273,11 @@ struct DotOpMFMAConversionHelper {
         }
       }
     }
+
+    // move setprio after the first mfma in the group.
```

Comment:
Could you mention why doing this? Mention its benefits.

Reply:
Done